srcpkgs/taisei/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'taisei'
 pkgname=taisei
-version=1.3.2
-revision=2
+version=1.4.1
+revision=1
 build_style=meson
-hostmakedepends="pkg-config python3-docutils"
-makedepends="SDL2_mixer-devel freetype-devel libwebp-devel libzip-devel cglm
- opusfile-devel"
+hostmakedepends="pkg-config python3-docutils python3-zstandard"
+makedepends="SDL2_mixer-devel freetype-devel libwebp-devel libzip-devel cglm opusfile-devel"
```

Comment:
Did you actually change anything here? I would just leave it as is (also the new version goes over 80 colums)